apps/services/license-api/src/app/modules/license/clients/drivingLicense/drivingLicenseMapper.ts

```diff
@@ -0,0 +1,126 @@
+import {
```

Comment:
Shouldn't we use the same mappers for license-service and license-api?

Reply:
This'll be absorbed in #9718